xc/device.ml

```diff
@@ -1131,18 +1134,56 @@ module PCI = struct
     (* Sort into the order the devices were plugged *)
     List.sort (fun a b -> compare (fst a) (fst b)) pairs
 
-  let add ~xs pcidevs domid =
+  let _pci_add ~xc ~xs domid (pcidev, bdf) =
```

Comment:
Might be worth adding a comment that `pcidev` is a host address, and `bdf` is a guest one (or rename them such that this is obvious).